api/src/main/java/org/apache/gravitino/stats/StatisticValues.java

```diff
@@ -0,0 +1,222 @@
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.stats;
+
+import com.google.common.base.Preconditions;
+import java.util.List;
+import java.util.Map;
+import org.apache.gravitino.rel.types.Type;
+import org.apache.gravitino.rel.types.Types;
+
+/** A class representing a collection of statistic values. */
+public class StatisticValues {
+
+  private StatisticValues() {}
+
+  /**
+   * Creates a statistic value that holds a boolean value.
+   *
+   * @param value the boolean value to be held by this statistic value
+   * @return a BooleanValue instance containing the provided boolean value
+   */
+  public static BooleanValue booleanValue(boolean value) {
+    return new BooleanValue(value);
+  }
+
+  /**
+   * Creates a statistic value that holds a long value.
+   *
+   * @param value the long value to be held by this statistic value
+   * @return a LongValue instance containing the provided long value
+   */
+  public static LongValue longValue(long value) {
+    return new LongValue(value);
+  }
+
+  /**
+   * Creates a statistic value that holds a double value.
+   *
+   * @param value the double value to be held by this statistic value
+   * @return a DoubleValue instance containing the provided double value
+   */
+  public static DoubleValue doubleValue(double value) {
+    return new DoubleValue(value);
+  }
+
+  /**
+   * Creates a statistic value that holds a string value.
+   *
+   * @param value the string value to be held by this statistic value
+   * @return a StringValue instance containing the provided string value
+   */
+  public static StringValue stringValue(String value) {
+    return new StringValue(value);
+  }
+
+  /**
+   * Creates a statistic value that holds a list of other statistic values.
+   *
+   * @param <T> the type of the values in the list
+   * @param values the list of statistic values to be held by this statistic value
+   * @return a ListValue instance containing the provided list of statistic values
+   */
+  public static <T> ListValue<T> listValue(List<StatisticValue<T>> values) {
+    return new ListValue<T>(values);
+  }
+
+  /**
+   * Creates a statistic value that holds a map of string keys to other statistic values.
+   *
+   * @param values the map of string keys to statistic values to be held by this statistic value
+   * @return an ObjectValue instance containing the provided map of statistic values
+   */
+  public static ObjectValue objectValue(Map<String, StatisticValue> values) {
```

Comment:
Java will complain about removing the type for `StatisticValue` here, can you figure out a better solution?

Reply:
I addressed the warning.